Apps/WebClient/src/ClientApp/src/models/dateWrapper.ts

```diff
@@ -54,6 +54,10 @@ export class DateWrapper {
                     this._internal_date = DateTime.fromISO(param, {
                         zone: "utc",
                     }).setZone("America/Vancouver");
+                } else if (/-\d{2}:\d{2}$/.test(param)) {
```

Comment:
this regex check if the param ends with -dd:dd, for example, "2020-10-25T10:56:52.1890766-07:00" param will return true, but "2020-10-22T00:00:00" returns false.

Reply:
what do you think about trying to capture the time zone? if that information is there then we can assume is not only a date, but a date time

the -07:00 part

Reply:
@FuriousLlama the "-07:00" part can't tell us which timezone right?

Reply:
Per my discussion with @FuriousLlama, I have removed the zone from the code above